packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -2,22 +2,63 @@ import React, { FC, Fragment, useEffect, useState } from 'react';
 import { Trans } from 'react-i18next';
 import { useCountdown } from 'usehooks-ts';
 import { useActiveWalletAccount, useSettings, useVerifyEmail } from '@deriv/api-v2';
+import {
+    DerivLightIcEmailSentIcon,
+    DerivLightIcFirewallEmailPasskeyIcon,
+    DerivLightIcSpamEmailPasskeyIcon,
+    DerivLightIcTypoEmailPasskeyIcon,
+    DerivLightIcWrongEmailPasskeyIcon,
+} from '@deriv/quill-icons';
 import { PlatformDetails } from '../../features/cfd/constants';
 import useDevice from '../../hooks/useDevice';
-import ChangePassword from '../../public/images/change-password-email.svg';
+import ChangePasswordIcon from '../../public/images/change-password-email.svg';
 import { TPlatforms } from '../../types';
 import { platformPasswordResetRedirectLink } from '../../utils/cfd';
-import { WalletButton } from '../Base';
+import { WalletButton, WalletText } from '../Base';
 import { WalletsActionScreen } from '../WalletsActionScreen';
 import './SentEmailContent.scss';
 
 type SentEmailContentProps = {
     description?: string;
+    isChangePassword?: boolean; // NOTE: This prop is ONLY used for rendering different email icons between either Change Password/Forgot password email modal
     isInvestorPassword?: boolean;
```

Comment:
This is for a very specific design requirement, where for now we have to use the old icon in `Change Password` modal, but for the `Forgot password` modal we are using the new icon:

https://github.com/binary-com/deriv-app/assets/103016120/c55cbb5c-347b-4a04-91fb-b9b1f4358efa